web/index.php

```diff
@@ -31,7 +31,11 @@
 
 try {
     // set anti-session-riding token
-    setcookie("wikidot_token7", md5(rand(0, 10000)), null, '/');
+    setsecurecookie("wikidot_token7", md5(rand(0, 10000)), 0, '/', GlobalProperties::$SESSION_COOKIE_DOMAIN);
```

Comment:
Is `rand()` a CSPRNG? Also I feel this would be cleaner if the cookie value was created in a separate line.

Reply:
If this were for any other purpose I would agree with you, but the token7 is an arcane bit of crap that hopefully we can live without or at least better understand the reason for here soon, and we can either enhance it or remove it altogether.